core/core.go

```diff
@@ -418,7 +418,7 @@ func makeSmuxTransportOption(mplexExp bool) libp2p.Option {
 		ConnectionWriteTimeout: time.Second * 10,
 		KeepAliveInterval:      time.Second * 30,
 		EnableKeepAlive:        true,
-		MaxStreamWindowSize:    uint32(1024 * 512),
+		MaxStreamWindowSize:    uint32(1024 * 8192),
```

Comment:
```suggestion
		MaxStreamWindowSize:    uint32(8<<20),
```

I think it is much cleaner in showing it is 8MB.

Reply:
I've changed it to `8 * 1024 * 1024`. I'm really not a fan of `8<<20`.